bin/bootstrax

```diff
@@ -500,32 +492,92 @@ def kill_process(pid, wait_time=None):
         os.kill(pid, sig)
 
 
-def override_target(rd):
+def _remove_veto_from_t(targets, remove=('_mv', '_nv'), _flip=False):
+    """Remove veto(s) from targets"""
+    start = targets
+    remove = strax.to_str_tuple(remove)
+    for r in remove:
+        targets = [t for t in targets if (r not in t)]
+    if _flip:
+        targets = [t for i, t in enumerate(start) if not np.in1d(start, targets)[i]]
+    return strax.to_str_tuple(targets)
+
+
+def _keep_veto_from_t(targets, keep='_nv'):
+    """Remove non-veto(s) targets"""
+    targets = _remove_veto_from_t(targets, remove=keep, _flip=True)
+    if not len(targets):
+        targets = strax.to_str_tuple('raw_records')
+    return targets
+
+
+def infer_target(rd):
     """
     Check if the target should be overridden based on the mode of the DAQ for this run
     :param rd: rundoc
-    :return: override False for LED or raw_records
+    :return: dict with the targets and the targets for post processing
     """
+    _t = args.targets
+    _pp = args.post_process
+
     if args.fix_target:
-        return False
+        return {'targets': strax.to_str_tuple(_t), 'post_processing': strax.to_str_tuple(_pp)}
+
+    n_fails = rd.get('bootstrax', {}).get('n_failures', 0)
```

Comment:
In principle this could be a bit streamlined, because rundocs are guaranteed to have a `bootstrax` field.

Reply:
good catch, will change it